databricks_cli/dbfs/api.py

```diff
@@ -75,12 +80,29 @@ class DbfsErrorCodes(object):
     RESOURCE_DOES_NOT_EXIST = 'RESOURCE_DOES_NOT_EXIST'
     RESOURCE_ALREADY_EXISTS = 'RESOURCE_ALREADY_EXISTS'
     PARTIAL_DELETE = 'PARTIAL_DELETE'
+    TOO_MANY_REQUESTS = 'TOO_MANY_REQUESTS'
+
+
+def retry_429(func):
+    @retry(wait=wait_random_exponential(multiplier=EXPONENTIAL_BACKOFF_MULTIPLIER, 
```

Comment:
There is actually a way of raising the original exception instead of RetryError, which is not explicit for the users.
From the docs (https://tenacity.readthedocs.io/en/latest/#error-handling):
> While callables that “timeout” retrying raise a RetryError by default, we can reraise the last attempt’s exception if needed:

Reply:
The we can also update the `test_mkdirs_stop_retrying` test to assert on the right exception.

Reply:
+1, I think it's better to use the reraise option here so user's can see more detailed information about the 429.